portfolio/src/main/webapp/script.js

```diff
@@ -46,23 +66,32 @@ function getComments() {
         addComment(comment)
       );
     });
+    Object.keys(data).length > 0 ? toggleDeleteAllButton(true) : toggleDeleteAllButton(false);
   })
 }
 
 /* Creates an <li> element containing text. */
 function addComment(comment) {
   const commentItem = document.createElement("li");
+  const commentHeader = document.createElement("div");
   const commentName = document.createElement("p");
   const commentMessage = document.createElement("p");
+  const commentDelete = document.createElement("img");
```

Comment:
I wonder if creating some subroutines could clean this tedious element creation up a bit.

Reply:
To clean this up, I added a method that creates an element with the given element id/classes/event handlers/etc.